routes/index.js

```diff
@@ -25,14 +25,14 @@ module.exports = function routes(app, options, next) {
 
         if (pkg && env && version) {
           const status = await app.models[type][find]({ pkg, env, version });
-
+          if (!status) return res.status(404).json(`No ${type} exists for ${pkg}@${version} in ${env}`);
```

Comment:
Why is this one a string response, but Line 34 below is an Error object?

Reply:
my error will correct